tests/integration/api/controllers/login.spec.js

```diff
@@ -157,6 +162,31 @@ describe('login', () => {
         .then(() => loginWithData({ user: user.username, password }))
         .then(response => expectRedirectToPasswordReset(response));
     });
+    
+
+    it('should fail if sso user', async () => {
+      await setupOidcLoginSettings('clientId');
+
+      const newUser = {
+        username: user.username,
+        oidc: true,
+        place: {
+          type: 'health_center',
+          name: 'Online place',
+          parent: 'PARENT_PLACE'
+        },
+        contact: {
+          name: 'OnlineUser'
+        },
+        roles: ['district_admin'], 
+      };
+      await utils.request({ path: '/api/v2/users', method: 'POST', body: newUser });
```

Comment:
thought: regarding your question about which create user endpoint to use, I think you are correct that it does not matter for this test (as long as it works).  Typically, when I am writing tests, I try to use the latest version of the endpoint (since if we ever remove support for endpoints it will be the earlier versions), but functionally it makes no difference here.